_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation.vue

```diff
@@ -207,7 +223,11 @@
             <VueShowdown
               tag="p"
               class="mb-0"
-              :markdown="$t('smartShoppingCampaignCreation.inputFiltersHelper', ['//google.com'])"
+              :markdown="
+                $t('smartShoppingCampaignCreation.inputFiltersHelper', [
+                  '//google.com',
```

Comment:
Thanks to this diff, I see that a URL is wrong cc @lolath-presta & @amaury-hanser